### PR TITLE
[Merged by Bors] - chore: remove repeated variables

### PR DESCRIPTION
**Pull request by adomani**
---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/Logic/Relator.lean [hunk 1]
```diff
@@ -116,7 +116,7 @@ lemma rel_eq {r : α → β → Prop} (hr : BiUnique r) : (r ⇒ r ⇒ (·↔·)
 
 open Function
 
-variable {α : Type*} {r₁₁ : α → α → Prop} {r₁₂ : α → β → Prop} {r₂₁ : β → α → Prop}
+variable {r₁₁ : α → α → Prop} {r₁₂ : α → β → Prop} {r₂₁ : β → α → Prop}
```
Comment (grunweg) [inline Mathlib/Logic/Relator.lean:119]: For other reviewers: this is already declared on line 99.

File: Mathlib/Init/Algebra/Classes.lean [hunk 2]
```diff
@@ -57,7 +57,7 @@ instance (priority := 100) (α : Sort u) (lt : α → α → Prop) [IsStrictWeak
 
 section
 
-variable {α : Sort u} {r : α → α → Prop}
+variable {r : α → α → Prop}
```
Comment (grunweg) [inline Mathlib/Init/Algebra/Classes.lean:60]: This and the other one are already declared on line 28.

**Review by grunweg — COMMENTED** [review]
Thanks!
maintainer merge

**Comment by riccardobrasca** [comment]
Thanks!

bors d+

**Comment by adomani** [comment]
bors merge

**Pull request closed without merging**